components/Item/Card.vue

```diff
@@ -324,7 +322,7 @@ export default Vue.extend({
 }
 
 .card-content {
-  background-color: #{map-get($material-dark, 'menus')};
+  background-color: #{map-get($material-light, 'menus')};
```

Comment:
How does it look like on cards with dark image?

Reply:
It only shows up when the image and the blurhash isn't loaded.

Currently, it shows up transparent, because for some reason it's not picking up the variable, but the idea is to avoid having a super dark card in the middle of a light page.